common/src/main/java/com/sequenceiq/cloudbreak/client/CertificateTrustManager.java

```diff
@@ -32,55 +28,16 @@ public static HostnameVerifier hostnameVerifier() {
     }
 
     public static SSLContext sslContext() {
-        // Create a trust manager that does not validate certificate chains
-        TrustManager[] trustAllCerts = {trustEverythingTrustManager()};
         try {
-            // Install the all-trusting trust manager
-            SSLContext sc = SslConfigurator.newInstance().createSSLContext();
-            sc.init(null, trustAllCerts, new SecureRandom());
-            LOGGER.debug("Trust all SSL certificates has been installed");
-            return sc;
-        } catch (KeyManagementException e) {
+            SSLContext aDefault = SSLContext.getDefault();
+            LOGGER.debug("Default SSL context has been initialised");
+            return aDefault;
+        } catch (NoSuchAlgorithmException e) {
             LOGGER.error(e.getMessage(), e);
             throw new RuntimeException("F", e);
```

Comment:
Not added in this change but I suggest using some more expressive error message here. :smile: